UndertaleModCli/Program.cs

```diff
@@ -572,8 +578,15 @@ private void CliQuickInfo()
         Console.WriteLine($"{Data.Paths.Count} Paths, {Data.Scripts.Count} Scripts, {Data.Shaders.Count} Shaders");
         Console.WriteLine($"{Data.Fonts.Count} Fonts, {Data.Timelines.Count} Timelines, {Data.GameObjects.Count} Game Objects");
         Console.WriteLine($"{Data.Rooms.Count} Rooms, {Data.Extensions.Count} Extensions, {Data.TexturePageItems.Count} Texture Page Items");
-        Console.WriteLine($"{Data.Code.Count} Code Entries, {Data.Variables.Count} Variables, {Data.Functions.Count} Functions");
-        Console.WriteLine($"{Data.CodeLocals.Count} Code locals, {Data.Strings.Count} Strings, {Data.EmbeddedTextures.Count} Embedded Textures");
+        if (!Data.IsYYC())
+        {
+            Console.WriteLine($"{Data.Code.Count} Code Entries, {Data.Variables.Count} Variables, {Data.Functions.Count} Functions");
+            Console.WriteLine($"{Data.CodeLocals.Count} Code locals, {Data.Strings.Count} Strings, {Data.EmbeddedTextures.Count} Embedded Textures");
+        }
+        else
+        {
+            Console.WriteLine("Unknown amount of Code entries and Code locals");
+        }
```

Comment:
So, if it's YYC, then it also won't write the amount of strings and embedded textures.

Reply:
`Info` never printed the amount of strings/embedded textures, regardless of YYC or not. I have added those now.